tests/eve-payload-07-http-gap/test.yaml

```diff
@@ -14,39 +14,84 @@ checks:
       alert.signature_id: 1
 - filter:
     count: 1
+    version: 8.0
     match:
       event_type: alert
       alert.signature_id: 1
       payload_printable: "GET /1 HTTP/1.0\r\nUser-Agent: Mozilla\r\n\r\n"
       payload_length: 40
 - filter:
     count: 1
+    version: 7.0.7
+    lt-version: 8.0
```

Comment:
You do not need  lt-version: 8.0 here, do you ?

Reply:
Yes -- this check if for main-7.0.x with the backport so [7.0.7, 8.0)

Reply:
The test for 8.0 checks for the recently-added `payload_length` (which isn't in main-7.0.x)

Reply:
So, this check also works for 8, you do not need lt-version, right ?

Reply:
That's true but there is a filter check immediately preceding this one that is targeted for 8+

Reply:
Will update.